dev/CommonStyles/InkToolbar_themeresources.xaml

```diff
@@ -299,7 +301,7 @@
                             <VisualStateGroup x:Name="GlyphVisualStates">
                                 <VisualState x:Name="ShowExtensionGlyph">
                                     <VisualState.Setters>
-                                        <Setter Target="ExtensionGlyph.Opacity" Value="1" />
+                                        <Setter Target="ExtensionGlyph.Opacity" Value="0" />
```

Comment:
Is this glyph never supposed to be visible, then?

Reply:
Now we want to make it visible, earlier the ask was not to be visible. Will update this accordingly!